src/spaceone/dashboard/service/domain_dashboard_service.py

```diff
@@ -164,7 +175,7 @@ def delete_version(self, params):
 
         return self.version_mgr.delete_version(domain_dashboard_id, version, domain_id)
 
-    @transaction(append_meta={'authorization.scope': 'DOMAIN_OR_USER'})
+    @transaction(append_meta={'authorization.scope': 'DOMAIN'})
     @check_required(['domain_dashboard_id', 'version', 'domain_id'])
     def revert_version(self, params):
```

Comment:
Requires permission validation logic similar to get and delete methods.
First, search for domain_dashboard_vo and verify the same.